### ISSUE TITLE
Homepage: Images missing alt text or insufficient alt text

### ISSUE THREAD

**Issue by andrewmarino** [unitymakesus/justgetsimple#51]
Photographs and images should have descriptive alt text, describing their subject(s).

Example: "Cemetery Before photo" should be more descriptive, such as "Oberlin's Historic Cemetery with headstones covered by weeds and overgrowth before renovation."

Example: "Cemetery After photo" should be more descriptive, such as "Oberlin's Historic Cemetery after renovation efforts. All overgrowth and debris has been removed, revealing the previously hidden original graves and headstones underneath."

**Comment by JacquelineOmollo** [comment]
Done

**Comment by andrewmarino** [comment]
There's a few new images missing their alt text.

**Comment by andrewmarino** [comment]
- [ ] Oberlin: A Village Rooted in Freedom (needs an alt text update)
- [ ] Upcoming Events
- [ ] In The News
- [ ] Oberlin's Historic Cemetery (before)
- [ ] Plummer T. Hall Restoration (before, and after images)
- [ ] Oberlin Historic Overlay District [Oberlin HOD] (describe map in alt text)
